test/1_stdlib/PrintFloat.swift

```diff
@@ -0,0 +1,202 @@
+// RUN: rm -rf %t && mkdir %t
+// RUN: %target-build-swift -emit-library -Xfrontend -enable-resilience -c  %S/Inputs/PrintTestTypes.swift -o %t/PrintTestTypes.o
+// RUN: %target-build-swift -emit-module -Xfrontend -enable-resilience -c  %S/Inputs/PrintTestTypes.swift -o %t/PrintTestTypes.o
+// RUN: %target-build-swift %s -Xlinker %t/PrintTestTypes.o -I %t -L %t -o %t/main
+// RUN: %target-run %t/main env %s
+// RUN: %target-run %t/main ru_RU.UTF-8 %s
+// REQUIRES: executable_test
+// XFAIL: linux
+
+import Swift
+import StdlibUnittest
+import Darwin
+import PrintTestTypes
+
+let PrintTests = TestSuite("PrintFloat")
+
+let arg = Process.arguments[1]
+if arg == "env" {
+  setlocale(LC_ALL, "")
+} else {
+  expectEqual("ru_RU.UTF-8", arg)
```

Comment:
I'm sorry, but are you sure this works with StdlibUnittest?  I don't think it passes down the arguments to the test runner process.  You would need to modify StdlibUnittest to do so, and here you would search for the argument anywhere among the command line parameters.